src/QMCHamiltonians/MomentumEstimator.cpp

```diff
@@ -101,8 +104,8 @@ void MomentumEstimator::registerCollectables(std::vector<ObservableHelper>& h5de
     //descriptor for the data, 1-D data
     std::vector<int> ng(1);
     //add nofk
-    ng[0] = nofK.size();
```

Comment:
Is this thre reason of CI failure?

Reply:
Thanks for the catch @ye-luo .  I got a little happy with ctrl-k